stdlib/2/SocketServer.pyi

```diff
@@ -39,8 +39,7 @@ class BaseServer:
         def __exit__(self, exc_type: Optional[Type[BaseException]],
                      exc_val: Optional[BaseException],
                      exc_tb: Optional[types.TracebackType]) -> bool: ...
-    if sys.version_info >= (3, 3):
```

Comment:
This file is identical to 3/sockerserver.pyi and this is enforced by check_consistent.py. Personally, I'd let them evolve separately, since the stubs can not be merged due to different names, and the Python 2 module will never change as opposed to other examples of check_consistent, where usually a third-party module is involved.

Reply:
I see. In that case, I think it's better to keep this code as is.

We want to keep the two files identical so that any improvements we make to one also get applied to the other (for example, if we discover that one of the argument types for an existing method is wrong).